velox/functions/sparksql/Hash.cpp

```diff
@@ -347,11 +360,18 @@ std::shared_ptr<exec::VectorFunction> makeHash(
 }
 
 std::vector<std::shared_ptr<exec::FunctionSignature>> xxhash64Signatures() {
-  return {exec::FunctionSignatureBuilder()
-              .returnType("integer")
-              .argumentType("any")
-              .variableArity()
-              .build()};
+  return {
+      exec::FunctionSignatureBuilder()
+          .returnType("integer")
+          .argumentType("any")
+          .variableArity()
+          .build(),
+      exec::FunctionSignatureBuilder()
+          .returnType("integer")
+          .constantArgumentType("bigint")
```

Comment:
Why would seed be integer for one function and bigint for another. Is this correct?

Reply:
In sparksql functions, the types of expression and seed in XxHash64 are `Long`, and in Murmur3Hash are `Int`. 
Murmur3Hash: https://github.com/apache/spark/blob/3a9185964a0de3c720a6b77d38a446258b73468e/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/hash.scala#L614C7-L614C7
XxHash64: 
https://github.com/apache/spark/blob/3a9185964a0de3c720a6b77d38a446258b73468e/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/hash.scala#L658

Reply:
Got it. Thank you for clarifying. It would be nice to document input types of these functions in https://facebookincubator.github.io/velox/functions/spark/binary.html